### PR TITLE
Update resource quota

### PR DESCRIPTION
**Pull request by loganhz**
https://github.com/rancher/rancher/issues/15339
https://github.com/rancher/rancher/issues/15333

### CODE REVIEW

File: app/components/project-resource-quota/component.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      value = `${ value }Mi`;
+    } else if ( key === 'requestsStorage' ) {
+      value = `${ value }Gi`;
+    }
+
+    return value;
+  },
+
+  convertToLimit(key, value) {
+    if ( !value ) {
```
Comment (westlywright) [inline app/components/project-resource-quota/component.js]: I think this function would be more readable (esp if we add more conditions) as a switch statement.

File: app/components/namespace-quota-row/template.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<td class="pr-10">
+  <div style="height: 35px;">
```
Comment (westlywright) [inline app/components/namespace-quota-row/template.hbs]: Single inline styles are fine every once in a while but 35px is used multiple times in this PR, move to a class.

File: app/components/project-quota-row/template.hbs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<td class="pr-10">
+  <div style="height: 35px;">
```
Comment (westlywright) [inline app/components/project-quota-row/template.hbs]: Single inline styles are fine every once in a while but 35px is used multiple times in this PR, move to a class.

File: app/components/project-resource-quota/component.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        resourceQuota:                 { limit },
+        namespaceDefaultResourceQuota: { limit: nsDefaultLimit },
+      }
+    }
+
+    this.sendAction('changed', out);
+  }),
+
+  convertToString(key, value) {
+    if ( !value ) {
```
Comment (westlywright) [inline app/components/project-resource-quota/component.js]: I think this function would be more readable (esp if we add more conditions) as a switch statement.

File: app/models/namespace.js [hunk 5]
```diff
@@ -2,10 +2,28 @@ import { computed, get } from '@ember/object';
...
 import Resource from 'ember-api-store/models/resource';
 import { parseExternalId } from 'ui/utils/parse-externalid';
+import { convertToMillis } from 'shared/utils/util';
+import { parseSi } from 'shared/utils/parse-unit';
 import C from 'ui/utils/constants';
 import { hasMany, reference } from 'ember-api-store/utils/denormalize';
 import StateCounts from 'ui/mixins/state-counts';
 
+export function convertResourceQuota(key, value) {
+  let out;
```
Comment (westlywright) [inline app/models/namespace.js]: I think this function would be more readable (esp if we add more conditions) as a switch statement.

**Comment by loganhz** [comment]
@westlywright Done. Thanks!

**Pull request merged**